_posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title: "A time travel through our latest Hackathon"
+date: 2019-04-01
+header_image: public/loc-interview.jpg
+header_position: center
+header_overlay: true
+category: events
+tags: ["hackathon", "company", "company culture"]
+authors: ["Christina"]
+about_authors: ["cgebken"]
+---
+
+The Hackathon is a recurring entry in our calendars.
+Each year we plan two days filled with interesting topics and lots of fun.
+But what does it really mean to take part in our Hackathon?
+Let me take you on a little time journey through the last month to give you some insights into the preparation and the event itself.
+
+## Three weeks ago
+
+It's getting serious guys.
+We were asked to brainstorm about possible topics and to add them to a Confluence list.
+These questions swirled through our minds:
+- What would help us in our everyday office life?
+- Which topic are on our I-always-wanted-to-work-on-that list?
+- Which topics are great but didn't make it yet and might never make it to one of the teams' sprints?
+
+My team member Birgit and I also brainstormed and added a topic to the list: Improving the [About us page](/about/) of the DevPortal.
+
+## Two weeks ago
+
+All topics were set and added to the list.
+Now, it was time to decide which topic we'd like to join.
+Let me tell you, it was a tough decision.
+So many interesting but different topics.
+Some of them aimed to improve our processes and workflows, such as a standardized on-boarding process, unified PR pipelines, a performance test, or a Circle CI wallboard.
+Others had a closer look at our products and merchants, e.g. a UX check for integrating apps, an app for importing products via CSV, or an instore checkout.
+
+These questions helped me to decide which topic I'd like to join:
+- How can I support the team that is working on that topic?
+- What will I learn when participating?
+- Which team members will I have? 
+
+Especially the last point was one of the most important ones for me.
```

Comment:
Hm, don't know if that's a bit toooo people-picky :D ?
Maybe then better combine these two sentences "as a hackathon is the "